packages/fluent-editor/src/modules/ai/index.ts

```
@@ -14,26 +29,11 @@ import {
   REPLACE_SELECT_ICON,
   RIGHT_ARROW_ICON,
   SEND_BTN_ICON,
+  SHARE_ICON,
```

ESLint

[error] 'SHARE_ICON' is defined but never used.

(unused-imports/no-unused-imports)